pages/jobs/index.js

```diff
@@ -35,19 +47,52 @@ export default function JobsIndexPage() {
             <p className="mt-3 max-w-2xl mx-auto text-xl leading-7 text-gray-500 sm:mt-4">
               Want to work on a great open-source project with a lovely team and a vibrant community? Come work with us!
             </p>
+            <p className="max-w-2xl mx-auto text-md leading-7 text-gray-400">
+            <span role="img">💡</span> Want to post a job? All job posting must be AsyncAPI-related jobs. <a className="ml-1 text-primary-500 hover:text-primary-400" href={`https://github.com/asyncapi/website/new/master/pages/jobs?value=${template}`} target="_blank" rel="noreferrer">Get started now!</a>
+            </p>
+            <p className="max-w-2xl mx-auto text-md leading-7 text-gray-400">Do you want to discuss about your job posting?
+            <a className="ml-1 text-primary-500 hover:text-primary-400" href="https://github.com/asyncapi/website/issues/new" target="_blank" rel="noreferrer">Get started here</a>
+            </p>
           </div>
-          <div className="mt-8 flex flex-col items-stretch sm:rounded-md">
-            <ul className="bg-white shadow overflow-hidden divide-y divide-gray-200 sm:w-2/3 sm:self-center">
-            {
-              posts.map((post, index) => (
-                <JobPostItem key={index} job={post} />
-              ))
-            }
-            </ul>
+          <div className="text-center">
+            {emptyChecker(posts) ? <div className="flex content-center justify-center">
+             <div>
+             <Empty/>
+             <p className="mt-3 max-w-2xl mx-auto text-xl leading-7 text-gray-500">No open positions currently. Check back later</p>
+             </div>
+            </div> : <div className="mt-8 flex flex-col items-stretch sm:rounded-md text-left">
+              <div className="divide-y divide-gray-200 sm:w-2/3 sm:self-center">
+                <Filter data={navItems} onFilter={onFilter} />
+              </div>
+              <ul className="bg-white shadow overflow-hidden divide-y divide-gray-200 sm:w-2/3 sm:self-center">
+                {
+                posts.map((post, index) => (
+                  <JobPostItem key={index} job={post} />
+                ))}
+              </ul>
+            </div>}
           </div>
         </div>
       </div>
       <Footer />
     </div>
   )
+}
+
+// eslint-disable-next-line no-unused-vars
+export async function getStaticProps(context) {
```

Comment:
do we need this helper with all those eslint rules disabled? Is it only to read encoded template? why not just having encoded template as a variable directly in code?

Reply:
Hmmm... That's thoughtful of you. I thought it will be too long but yeah I'll do that.